isis/appdata/import/CassiniISS.tpl

```diff
@@ -172,22 +172,26 @@ ObservationId = {{ OBSERVATION_ID.Value }}
 ProductId     = {{ ProductId }}
 {% endblock%}
 
-{% block bandbin %}
-{{ super() }}
-FilterName   = {{ filterName.0 }}/{{ filterName.1 }}
-OriginalBand = 1
-{% set cassiniIssBandInfo = CassiniIssBandInfo(INSTRUMENT_ID.Value, filterName.0, filterName.1) %}
-Center       = {{ cassiniIssBandInfo.0 }}
-Width        = {{ cassiniIssBandInfo.1 }}
-{% endblock%}
+{% block additional_groups %}
+Group = BandBin
+  {% if exists("FILTER_NAME") %}
+  {% set filterName=FILTER_NAME.Value %}
+  FilterName   = {{ filterName.0 }}/{{ filterName.1 }}
```

Comment:
should this `{{ filterName.0 }}/{{ filterName.1 }}` be this `{{ filterName.0 }}{{ filterName.1 }}`?

Reply:
The desired output is `CO-S-ISSNA/ISSWA-2-EDR-V1.0` so that is why we have a dash.